SPECS/KeysInUse-OpenSSL/KeysInUse-OpenSSL.spec

```diff
@@ -0,0 +1,83 @@
+Summary:        The KeysInUse Engine for OpenSSL allows the logging of private key usage through OpenSSL
+Name:           KeysInUse-OpenSSL
+Version:        0.3.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Libraries
+URL:            https://github.com/microsoft/KeysInUse-OpenSSL
+#Source0:       https://github.com/microsoft/KeysInUse-OpenSSL/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  golang >= 1.16.6
+BuildRequires:  make
+BuildRequires:  openssl-devel
+Requires:       openssl < 1.1.2
+Requires:       openssl >= 1.1.1
+
+%description
+ The KeysInUse Engine for OpenSSL allows the logging of private key usage through OpenSSL
+
+%prep
+%setup -q
+
+%build
+export GO111MODULE=off
+
+cmake -DCMAKE_TOOLCHAIN_FILE=./cmake-toolchains/linux-amd64-glibc.cmake -H./ -B./build
+cmake --build ./build --target keysinuse
+
+cd ./packaging/util
+make $(realpath ../../bin/keysinuseutil)
+
+%define keysinuse_dir %{buildroot}/%{_libdir}/keysinuse/
+
+%install
+mkdir -p %{keysinuse_dir}
+mkdir -p %{buildroot}%{_bindir}/
+
+cp ./bin/keysinuse.so %{keysinuse_dir}
```

Comment:
You may want to use install -m XXX instead, since it ensures that the intended permissions are applied. Example:  https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/bash/bash.spec#L57